ntoskrnl/io/pnpmgr/pnpmgr.c

```diff
@@ -1770,6 +1849,7 @@ IopQueryHardwareIds(PDEVICE_NODE DeviceNode,
    UNICODE_STRING ValueName;
    NTSTATUS Status;
    ULONG Length, TotalLength;
+   BOOLEAN IsValideID;
```

Comment:
s/Valide/Valid/

Reply:
?

Reply:
I mean it should be "IsValidID" without and e.

Reply:
Yeah, it's so much clearer )
Changed